keras/utils/data_utils.py

```diff
@@ -471,6 +478,7 @@ def __init__(self, sequence,
                  use_multiprocessing=False,
                  shuffle=False):
         self.sequence = sequence
+        self.uuid = int(np.random.rand(1)[0] * 255)
```

Comment:
`np.random.randint(0, 256)` would be more readable imo -- but that isn't actually a UUID unless you check for unicity. Maybe better to index the sequences in order instead of with random integers?